src/commands/trace/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+import {AxiosPromise, AxiosResponse} from 'axios'
+
+export interface CustomIDs {
+  id: string
+  parent_id?: string
+}
+
+export interface Payload {
+  command: string
+  custom: CustomIDs
+  data: Record<string, string>
+  end_time: string
+  is_error: boolean
+  name: string
```

Comment:
name here is mapped to the span's operation name? I'd update the `name` to make it more explicit

Reply:
The decision on how to map this is a backend decision. At the datadog-ci level, the user can provide a human-readable name to represent the span, which is what is represented by this field. Does that make sense?

(At the moment, this value is mapped to the `resource` value of the span)

Reply:
yeah it makes sense, I just worry that `name` is not meaningful enough. It's minor though, we can keep it as is